content/docs/start/data-and-model-versioning.md

```diff
@@ -223,25 +219,25 @@ $ dvc checkout
 
 <details>
 
-### ⚙️ Expand to get the previous version of the dataset.
+### ⚙️ Expand to get the previous version of the dataset 🕑
 
-Let's cleanup the previous artificial changes we made and get the previous :
+Let's go back to the original version of the data:
 
 ```dvc
-$ git checkout HEAD^1 data/data.xml.dvc
+$ git checkout HEAD~1 data/data.xml.dvc
 $ dvc checkout
 ```
 
-Let's commit it (no need to do `dvc push` this time since the previous version
-of this dataset was saved before):
+Let's commit it (no need to do `dvc push` this time since this original version
+of the dataset was already saved):
 
 ```dvc
 $ git commit data/data.xml.dvc -m "Revert dataset updates"
 ```
 
 </details>
 
-Yes, DVC is technically not even a version control system! `.dvc` files content
+Yes, DVC is technically not even a version control system! `.dvc` files' content
 defines data file versions. Git itself provides the version control. DVC in turn
```

Comment:
👍  How about "`.dvc` file contents define the versions of data files" ?

Reply:
changed